worker-core/src/main/java/com/hpe/caf/worker/core/WorkerTaskImpl.java

```diff
@@ -765,4 +765,21 @@ private static int getBufferLimit()
         return bufferLimit;
     }
 
+    void sendPoisonMessage(final String workerIdentifier)
```

Comment:
- Why is this package-private?
- Why is it here?  Can't it just be a private utility function in `StreamingWorkerWrapper`?